### ISSUE TITLE
Value List - Avoid inappropriate use of ARIA roles, states, and properties - (2036570519)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7848]
### Violation: 
 Avoid inappropriate use of ARIA roles, states, and properties 

![image](https://github.com/Esri/calcite-design-system/assets/5023024/a046ece4-5f90-42fd-b095-31434cf2ad1e)

### WCAG Reference: 
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Severity: 
 5 

### Media Type: 
 ARIA 

--- 
### Areas for Remediation: 
- Title: Calcite Panel - Avoid inappropriate use of ARIA roles, states, and properties - (2036570519) 
- Module: Calcite Panel 
**Issue**
There are <calcite-value-list> elements that make use of ARIA roles menu inappropriately. Examples include:

- Events, Data, Documents, Apps & Maps, Custom catalog 1
- Initiatives

**User Impact**
When inappropriate ARIA roles, states, and/or properties are used, screen reader users will mistake these elements for other control structures or the use may prevent users of assistive technology from interacting with the control.

**Code Reference** 
 ```
<calcite-value-list drag-enabled="" aria-busy="false" role="menu" calcite-hydrated="">
  <calcite-value-list-item label="Events" value="events" id="calcite-value-list-item-10547a8a-04b5-8c9e-5387-af6dffc83893" icon="grip" calcite-hydrated="">
    (...)
   </calcite-value-list-item>
</calcite-value-list>
 ``` 
**Suggestion**
Developers must avoid inappropriate use of ARIA roles, states, and/or properties. Remove role="menu" from ``` <calcite-value-list> ``` elements.

**Compliant Code Example** 
 ```
<calcite-value-list drag-enabled="" aria-busy="false" calcite-hydrated="">
  <calcite-value-list-item label="Events" value="events" id="calcite-value-list-item-10547a8a-04b5-8c9e-5387-af6dffc83893" icon="grip" calcite-hydrated="">
    (...)
   </calcite-value-list-item>
</calcite-value-list>
 ```
 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by driskull** [comment]
@geospatialem is this for value-list or panel?

**Comment by geospatialem** [comment]
> @geospatialem is this for value-list or panel?

Value List, this was one of the more complex examples and identified incorrectly in the report.

**Comment by driskull** [comment]
Can we close issues for components that are deprecated?

**Comment by geospatialem** [comment]
Closing as the component is deprecated and additional a11y efforts will be spent on the `list` components.
